roles/build_containers/templates/build_containers.sh.j2

```diff
@@ -49,7 +49,7 @@ openstack tcib container image build \
 {%   endfor %}
 {%   endif %}
      --tcib-extra tcib_release={{ ansible_distribution_major_version }} \
-     --tcib-extra tcib_python_version={{ (ansible_distribution_major_version is version('9', '<')) | ternary ('3.6', '3.9') }} \
+     --tcib-extra tcib_python_version={{ (ansible_distribution_major_version is version('10', '<')) | ternary ('3.9', '3.12') }} \
```

Comment:
Tnis change is unrelated to your PR. Please remove it.

Reply:
Note this role is used in RHOSO-18 where python3.9 is used. I would drop 3.6 completely(No longer used by RHOSO anywhere) and keep 3.9 and 3.12 only.

Reply:
OK, makes sense.